.claude/hooks/autoapprove.py

```
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import json
+import os
+import sys
+
+
+def main() -> int:
+    try:
+        raw = sys.stdin.read()
+        request = json.loads(raw)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)